dbt-cta/echo19/models/0_ctes/text_contact_ab1.sql

```diff
@@ -0,0 +1,113 @@
+-- SQL model to build a hash column based on the values of this record
+-- depends_on: {{ source('cta', '_stg_text_contact') }}
+
+select
+    textResultId,
+    scheduleId,
+    programId,
+    programName,
+    scheduleName,
+    "from",
+    "to",
```

Comment:
oooh good catch! I added the weirdness to the other file because BQ was complaining about the syntax... not sure why it didn't complain here. I'll update for consistency and make sure it runs ok

Reply:
oh wait nvm, this doesn't need to be changed in the _ab1 file. BigQuery complains that columns need names but that complaint is only based on the final query - these ones are rendered as CTEs that roll into the final query that creates the _base table, so only the _base model SQL needs to rename the columns